akka-projection-jdbc/src/it/resources/db/oracle-init.sql

```diff
@@ -0,0 +1,2 @@
+CREATE USER "test_schema" IDENTIFIED BY password;
+GRANT UNLIMITED TABLESPACE TO "test_schema"
```

Comment:
Don't we need a `;` at the end of this line?

Reply:
Test would fail if that command was not executed, but I'll add it for consistency anyway.

Reply:
Done.